pkg/sriov/sriov.go

```diff
@@ -126,7 +126,8 @@ type Manager interface {
 	SetupVF(conf *sriovtypes.NetConf, podifName string, cid string, netns ns.NetNS) (string, error)
 	ReleaseVF(conf *sriovtypes.NetConf, podifName string, cid string, netns ns.NetNS) error
 	ResetVFConfig(conf *sriovtypes.NetConf) error
-	ApplyVFConfig(conf *sriovtypes.NetConf) error
+	ApplyVFConfig(conf *sriovtypes.NetConf, pfLink netlink.Link) error
```

Comment:
why did you change this interface method ?  just to avoid getting netlink handle twice ?
I think it should be kept as is for the sake of simplicity as before